### ISSUE TITLE
Remain logged in across restarts

### ISSUE THREAD

**Issue by mrbiber** [turtl/desktop#11]
For people with an encrypted HDD, an option to remain logged in across app restarts would sacrifice no security and enhance usability.

**Comment by gregf** [comment]
:+1:

**Comment by orthecreedence** [comment]
Not a bad idea! I'll try and fit this into the next desktop release.

**Comment by onlygecko** [comment]
Is there any update on the 'stay logged in' feature for the desktop app?

**Comment by gjelsas** [comment]
this would be great, and encouraging to using a strong random password!

**Comment by Mikescher** [comment]
yeah that's pretty much the only reason I haven't migrated from Evernote yet - it's rather annoying to enter your pasword on every startup if you have no critical information stored

**Comment by Viperoo** [comment]
Perhaps only the last login?

**Comment by jplatte** [comment]
Is there a workaround or something? Otherwise I'll have to change to a weaker password because I'm definitely not launching my password manager for this every time.

**Comment by DoTheEvo** [comment]
I am not using a note application that I have to log in every time I launch a desktop application on my PC. At least username could be remembered between restart.

**Comment by fred9176** [comment]
+1 for me. This app is useless if credentials are not stored between each lauches. Most apps (online and desktop) allow to save password or not.
